lms/models/lti_user.py

```diff
@@ -44,9 +49,11 @@ def is_learner(self):
         return "learner" in self.roles.lower()
 
     @staticmethod
-    def from_auth_params(application_instance, lti_core_schema):
+    def from_auth_params(request, application_instance, lti_core_schema):
```

Comment:
This method itself could become "unserlize" but it will take more work for clients that already use it. I reckon we should keep it for now.